tests/unit/core/reducers/test_addons.js

```diff
@@ -105,4 +106,24 @@ describe('addon reducer', () => {
       baz: { description: null },
     });
   });
+
+  it('pulls down the "is_restart_required" attribute from the file', () => {
```

Comment:
This is a nitpick but `reads `is_restart_required` attribute from the file` might be a bit clearer? I dunno, "pulls down" seems overly verbose 😄